### PR TITLE
make docker build working with after a fresh checkout

### PR DESCRIPTION
**Pull request by zerdos**

### CODE REVIEW

File: build/tasks.ts [hunk 1]
```diff
@@ -224,7 +224,7 @@ const ensureCloned = register("vscode:clone", async (runner) => {
 	}
 
 	runner.cwd = vscodePath;
-	const checkout = await runner.execute("git", ["checkout", "tags/1.31.1"]);
+	const checkout = await runner.execute("git", ["checkout", "tags/1.31.1"]); //TODO: this tag should come from a parameter
```
Comment (kylecarbs) [inline build/tasks.ts]: What do you mean by a parameter?

Comment (nhooyr) [reply build/tasks.ts]: I guess he means it shouldn't be hard coded.

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM node:8.15.0
+FROM node:8.9.3
```
Comment (nhooyr) [inline Dockerfile]: Why the change?

File: Dockerfile [hunk 3]
```diff
@@ -18,7 +18,7 @@ RUN yarn && yarn task build:server:binary
 # We deploy with ubuntu so that devs have a familiar environemnt.
 FROM ubuntu:18.10
 WORKDIR /root/project
-COPY --from=0 /src/packages/server/cli-linux /usr/local/bin/code-server
+COPY --from=0 /src/packages/server/cli-linux-x64   /usr/local/bin/code-server
```
Comment (nhooyr) [inline Dockerfile]: afaik, there shouldn't be a `-x64` suffix?

@kylecarbs

**Comment by nhooyr** [comment]
I don't understand what this PR fixes. How does `docker build .` not work after a fresh checkout?

File: Dockerfile (hunk 3 above)
Comment (zerdos) [reply Dockerfile]: There should be - that is actually the only output

**Comment by nhooyr** [comment]
See #72

**Comment by zerdos** [comment]
This PR makes the Dockerfile actually build without error :)

File: Dockerfile (hunk 3 above)
Comment (nhooyr) [reply Dockerfile]: Yup. This was recently changed without me knowing. It's fixed in #109.
